src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -234,4 +234,16 @@
   <data name="PX1026Fix" xml:space="preserve">
     <value>Remove underscores from the DAC declaration</value>
   </data>
+  <data name="PX1027Fix" xml:space="preserve">
+    <value>Remove forbidden field from the DAC declaration</value>
+  </data>
+  <data name="PX1027Title" xml:space="preserve">
+    <value>The field {0} cannot be declared within DAC declarations</value>
+  </data>
+  <data name="PX1028Fix" xml:space="preserve">
+    <value>Remove all constructors entirely</value>
```

Comment:
@saloponov, Doesn't your codefix remove only one constructor with code fix - the one with diagnostic? If so, then this message is not correct here